docs/resources/dw_aws_cluster.md

```diff
@@ -1,7 +1,7 @@
 ---
 # generated by https://github.com/hashicorp/terraform-plugin-docs
 page_title: "cdp_dw_aws_cluster Resource - terraform-provider-cdp"
-subcategory: "Data Warehouse"
+subcategory: ""
```

Comment:
this requires another change because modifying the `.md` files would lead to continuous overwrite since they're generated from the templates, thus the templates needs to be updated to have a permanent effect on the `md`s. I've mentioned it to @vcsomor

Reply:
this has been fixed, please use templates

Reply:
use `make build docs` to generate the new documentation

Reply:
Thanks for the review @tevesz , @gregito and @vcsomor . I have added all the fixes

Reply:
@prateek1192 subcategory is still incorrect in the latest patch, you didn't add a template: https://github.com/cloudera/terraform-provider-cdp/tree/main/templates/resources